MessageTerminal/MessageTerminal.cxx

```diff
@@ -99,7 +99,6 @@ void get_status(istringstream& iss)
                 case STATUS_IDLE:   puts("Status: IDLE"); break;
                 case STATUS_MOVING: puts("Status: MOVING"); break;
                 case STATUS_HOMING: puts("Status: HOMING"); break;
-                case STATUS_FAULT:  puts("Status: FAULT"); break;
                 default:            puts("ERR: Invalid status"); break;
```

Comment:
What's the reason for removing the fault status?

Reply:
Oops I removed it then added it back everywhere except here. Will add it back here as well.

Reply:
Will address this in future PR